client/src/components/ModalView.jsx

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import React from 'react';
+import { useState } from 'react';
+import { Modal, Button, Space, Input } from 'antd';
+import { ExclamationCircleOutlined } from '@ant-design/icons';
+
+type ModalProps = {
+  isAddCategory: boolean,
+  isDeleteCategory: boolean,
+};
+
+function ModalView(props: ModalProps) {
+  const { isAddCategory, isDeleteCategory } = props;
+
+  const [isModalVisible, setIsModalVisible] = useState(false);
+
+  const showModal = () => {
+    setIsModalVisible(true);
+  };
+
+  const handleOk = () => {
+    setIsModalVisible(false);
+  };
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+
+  function DeleteCategory() {
```

Comment:
Super small nit - to stay consistent with the practices in this file, can we use camel case with the function name (deleteCategory vs DeleteCategory), and use the `const deleteCategory = ()` notion?